.github/workflows/build_wheels.yml

```diff
@@ -0,0 +1,128 @@
+    steps:
+      - uses: actions/checkout@v2.3.4
+        with:
+          submodules: "recursive"
+
+      - name: Build wheels
+        uses: joerick/cibuildwheel@v1.10.0
+        env:
+          # Build for cpython >= 3.6.
+          CIBW_PROJECT_REQUIRES_PYTHON: "==${{ matrix.pyver }}.*"
+          CIBW_SKIP: pp*
+
+          # Build only on 64-bit architectures.
+          CIBW_ARCHS_MACOS: x86_64
+          CIBW_ARCHS_LINUX: auto64
+          CIBW_ARCHS_WINDOWS: auto64
+
+          # Configure environment variables.
+          CIBW_ENVIRONMENT_LINUX: "CMAKE_PREFIX_PATH=/project/tbb LD_LIBRARY_PATH=/project/tbb/lib/intel64/gcc4.8:$LD_LIBRARY_PATH"
+          CIBW_ENVIRONMENT_MACOS: "CMAKE_PREFIX_PATH=/Users/runner/work/freud/freud/tbb LD_LIBRARY_PATH=/Users/runner/work/freud/freud/tbb/lib/intel64/gcc4.8:$LD_LIBRARY_PATH"
+
+          # Set up TBB.
+          CIBW_BEFORE_BUILD_LINUX: "source .github/workflows/cibuildwheel-before-build.sh {package} linux"
+          CIBW_BEFORE_BUILD_MACOS: "source .github/workflows/cibuildwheel-before-build.sh {package} macos"
+          CIBW_BEFORE_BUILD_WINDOWS: "powershell .github/workflows/cibuildwheel-before-build.ps1 {package}"
+
+          # Test the wheels.
+          CIBW_BEFORE_TEST_LINUX: "source .github/workflows/cibuildwheel-before-test.sh {package}"
+          CIBW_BEFORE_TEST_MACOS: "source .github/workflows/cibuildwheel-before-test.sh {package}"
+          CIBW_BEFORE_TEST_WINDOWS: "powershell .github/workflows/cibuildwheel-before-test.ps1 {package}"
+          CIBW_TEST_COMMAND: "cd {package}/tests && pytest . -v --log-level=DEBUG -n auto"
+
+      - uses: actions/upload-artifact@v2.2.3
+        with:
+          name: dist
+          path: ./wheelhouse/*.whl
+
+  build_sdist:
+    name: Build source distribution
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2.3.4
+
+      - uses: actions/setup-python@v2.2.2
+        name: Install Python
+        with:
+          python-version: '3.9'
+
+      - name: Install build requirements
+        run: python3 -m pip --disable-pip-version-check install scikit-build cmake -r requirements.txt --upgrade --progress-bar=off
+
+      - name: Build sdist
+        run: python setup.py sdist
+
+      - uses: actions/upload-artifact@v2.2.3
+        with:
+          name: dist
+          path: dist/*.tar.gz
+
+  upload_pypi:
+    name: Publish [PyPI]
+    needs: [build_wheels, build_sdist]
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Download artifacts
+        uses: actions/download-artifact@v2.0.9
+        with:
+          name: dist
+          path: dist
+
+      - name: Check files
+        run: ls -lR dist
+
+      - name: Upload to PyPI
+        # upload to PyPI on every tag starting with 'v'
+        if: startsWith(github.ref, 'refs/tags/v')
+        uses: pypa/gh-action-pypi-publish@v1.4.2
+        with:
+          user: __token__
+          password: ${{ secrets.PYPI_API_TOKEN }}
+
+      - name: Upload to TestPyPI
```

Comment:
We seem to be uploading to testPyPI after we upload to PyPI. Should it be the reverse of that?

Reply:
The order shouldn't matter -- the conditions are different for each type of upload. Josh has tested this for GSD so I'm not concerned.